package.json

```diff
@@ -1,7 +1,17 @@
 {
```

Comment:
Check all dependencies to ensure they are only needed in the production build. Otherwise, move them to devDependencies.